pom.xml

```diff
@@ -1,30 +1,23 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.algolia</groupId>
-  <artifactId>algoliasearch-parent</artifactId>
-  <version>2.22.1-SNAPSHOT</version>
-  <packaging>pom</packaging>
+  <artifactId>algoliasearch</artifactId>
+  <version>3.0.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
 
-  <name>Algolia Search Parent POM</name>
+  <name>Algolia Search</name>
   <description>Java client for Algolia Search API</description>
   <url>https://github.com/algolia/algoliasearch-client-java-2</url>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-
-    <slf4j.version>1.7.25</slf4j.version>
-    <apache-http-client.version>4.5.5</apache-http-client.version>
-    <guava.version>24.0-jre</guava.version>
+    <async-http-client.version>4.1.4</async-http-client.version>
```

Comment:
Why the regression from `4.5.5` to `4.1.4`? :thinking:

Reply:
Because it's not the same `HttpClient` 😄 

- https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient (old one sync http client)
- https://mvnrepository.com/artifact/org.apache.httpcomponents/httpasyncclient (new one async http client)

Reply:
Haha the devil is in the details :upside_down_face: